lib/codebreaker/config/locales/en.yml

```diff
@@ -0,0 +1,38 @@
+en:
```

Comment:
all keys where u don't have specific symbols such as ```:``` or ```'```, u can use just:
  ```your_key: Simple text```, without wrapping in quotes

Reply:
fixed